lib/dogma/script.ex

```diff
@@ -8,6 +8,7 @@ defmodule Dogma.Script do
   alias Dogma.Rules
   alias Dogma.Script
   alias Dogma.Error
+  alias Dogma.Script.Skipper
```

Comment:
The name is super odd. It worried me a bit that I couldn't find a name. The following is what the module does:

Decides what files should be run based on a modules configuration.

That sounds to me like a discrete unit of work. Therefore suitable for a module. But I couldn't think of a good name. Any suggestions?